src/primitives/TreatmentProgress.js

```diff
@@ -0,0 +1,43 @@
+import React from "react"
+import styled from "styled-components"
+import { observer } from "mobx-react"
+
+import CircularProgressbar from "react-circular-progressbar"
+import { primary } from "../colors"
+
```

Comment:
I see, this is CustomProgressBar but with some refactoring? Looks a lot better

Reply:
Yep! Renamed the file, moved some styles into a `styled-component`.